### DISCUSSION TITLE
native popovers

### DISCUSSION THREAD

**Discussion by zewa666** [ghiscoding/slickgrid-universal#1709] [Ideas]
hey there @ghiscoding. while not ready for primetime due to lack of firefox and safari support it might still be worth to keep an eye on the new browser native features to create popovers/tooltips

https://www.voorhoede.nl/en/blog/the-popover-api-your-new-best-friend-for-tooltips/

**Comment by ghiscoding** [comment]
ah yeah I think I've seen a blog post on GitHub talking about popover but didn't pay much attention. According to [can I use](https://caniuse.com/?search=popover) website, it seems to be only IE11 and possible Opera Mini that aren't supported (or unknown in the case of Opera Mini, which is often used in less reliable telephony like Latin America). What will this give us anyway? I mean, the Custom Tooltip Component I have created seems to work nicely, I guess this popover could maybe lower build size, is that it? Maybe auto-anchor positioning could be interesting though.

I started another thing last night, I'll open the Pagination to be optionally provided as an External Component (not sure how that's gonna go with Angular, I might be able to reuse the dynamic comp for Row Detail feature), we'll see how long that takes

**Reply by ghiscoding** [reply]
ahh wait I see, it's the `position-anchor` that isn't supported in Firefox/Safari... I guess we won't be able to use this for a couple of years then.

**Reply by ghiscoding** [reply]
I found this [polyfill](https://github.com/oddbird/css-anchor-positioning) which might make this more usable today. Still unsure if that will bring us much benefit on size decrease though? Maybe auto-anchor positioning (with `position-try-fallbacks`) could be interesting though since I don't currently have that in my Custom Tooltip plugin.

some interesting youtube training
- https://www.youtube.com/watch?v=aCEtSEFEYQ0
- https://www.youtube.com/live/76hIB2L_vs4

**Reply by ghiscoding** [reply]
@zewa666 I added you as a Collaborator to the project, since you're pretty much part of the project team 😉 invite sent check your email
